### ISSUE TITLE
new issues

### ISSUE THREAD

**Issue by gaoshihao333** [SongtaoLiu0823/LAGNN#4]
cvae_pretrain.py 's  U_score = F.nll_loss(output[idx_train], labels[idx_train]) - cross_entropy / args.num_models ,what does it  mean?

**Comment by SongtaoLiu0823** [comment]
> cvae_pretrain.py 's U_score = F.nll_loss(output[idx_train], labels[idx_train]) - cross_entropy / args.num_models ,what does it mean?

It computes the u score based on the Eq.(10) of the paper https://arxiv.org/pdf/2109.03856.pdf
